.github/workflows/ci.yml

```diff
@@ -35,13 +35,8 @@ jobs:
         if: runner.os == 'linux'
 
       - name: Check the format
-        run: cargo fmt --all -- --check
-        if: runner.os == 'linux' && matrix.toolchain == 'stable'
-
-      # type complexity must be ignored because we use huge templates for queries
-      # -A clippy::manual-strip: strip_prefix support was added in 1.45. we want to support earlier rust versions
-      - name: Clippy
-        run: cargo clippy --all-targets --all-features -- -D warnings -A clippy::type_complexity -A clippy::manual-strip
+        # See tools/ci/src/main.rs for the commands this runs
+        run: cargo ci
```

Comment:
This makes it a bit slower to find the failed step.

Reply:
In what way?

Reply:
Ah, I see it. It took 38s to compile the xtask in [the first run](https://github.com/bevyengine/bevy/pull/1387/checks?check_run_id=1828137711#step:5:386), and 24s in [the second run](https://github.com/bevyengine/bevy/pull/1387/checks?check_run_id=1828156485#step:5:194).  So, that makes the `build (stable, ubuntu-latest)` job ~30s longer. Looking back at the 5 most recent passing builds before this, that is also typically the longest job (which makes sense -- it's the only one we do `cargo fmt` and `cargo clippy` in), so I expect this there will generally be a ~30s regression in exchange for the cross-platform portability, integration, and Rusty-ness of this approach. 😉